<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientController</class>
 <widget class="QWidget" name="ClientController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>601</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Ressources</string>
    </attribute>
    <widget class="QTabWidget" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>751</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ajouter client</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1081</width>
         <height>361</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(243, 224, 219);
color:black;
border-radius:15px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>20</y>
         <width>201</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#55aa7f;&quot;&gt;Add Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>170</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_004748227-removebg-preview (1).png</normaloff>../../Downloads/image_2023-10-05_004748227-removebg-preview (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_004636688-removebg-preview.png</normaloff>../../Downloads/image_2023-10-05_004636688-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/logooo.png</normaloff>:/new/prefix1/assets/logooo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>70</y>
         <width>231</width>
         <height>241</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 152);
color:black;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/logooo.png</normaloff>../../Desktop/logooo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_adresse">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>170</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>           Adress</string>
       </property>
      </widget>
      <widget class="QPushButton" name="on_pushButton_ajouter_clicked">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>320</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55aa7f;&quot;&gt;Sing-up&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(113, 200, 97);
color:black;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_courreil">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>140</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>            Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>             Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ptv">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>230</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>         Point_Vente</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ptl">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>260</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>       Point_Location</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>200</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>200</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-11-01_173739053-removebg-preview.png</normaloff>../../Downloads/image_2023-11-01_173739053-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="on_pushButton_ajouter_clicked_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>320</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55aa7f;&quot;&gt;Sing-up&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(113, 200, 97);
color:black;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>80</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>             ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_error">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>290</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red; background:grey</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>pushButton_2</zorder>
      <zorder>label_5</zorder>
      <zorder>pushButton</zorder>
      <zorder>pushButton_6</zorder>
      <zorder>lineEdit_adresse</zorder>
      <zorder>on_pushButton_ajouter_clicked</zorder>
      <zorder>lineEdit_courreil</zorder>
      <zorder>lineEdit_nom</zorder>
      <zorder>lineEdit_ptv</zorder>
      <zorder>pushButton_10</zorder>
      <zorder>pushButton_8</zorder>
      <zorder>pushButton_4</zorder>
      <zorder>lineEdit_ptl</zorder>
      <zorder>lineEdit_date</zorder>
      <zorder>pushButton_9</zorder>
      <zorder>on_pushButton_ajouter_clicked_2</zorder>
      <zorder>pushButton_11</zorder>
      <zorder>lineEdit_id</zorder>
      <zorder>label_error</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Supprimer client</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_19">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>140</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</normaloff>../../Downloads/image_2023-10-05_004909935-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_20">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-50</y>
         <width>1141</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(243, 224, 219);
color:black;
border-radius:15px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_21">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>110</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Downloads/image_2023-10-05_005051266-removebg-preview.png</normaloff>../../Downloads/image_2023-10-05_005051266-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline; color:#55aa7f;&quot;&gt;Delete Account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_25">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/logooo.png</normaloff>:/new/prefix1/assets/logooo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_26">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>60</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0, 0, 0, 152);
color:black;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../Desktop/logooo.png</normaloff>../../Desktop/logooo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="on_pushButton_supprimer_clicked">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55aa7f;&quot;&gt;Sing-up&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(113, 200, 97);
color:black;
border-radius:15px;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="llineEdit_IDS">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>              ID</string>
       </property>
      </widget>
      <zorder>pushButton_19</zorder>
      <zorder>pushButton_20</zorder>
      <zorder>label_7</zorder>
      <zorder>pushButton_25</zorder>
      <zorder>pushButton_26</zorder>
      <zorder>on_pushButton_supprimer_clicked</zorder>
      <zorder>llineEdit_IDS</zorder>
      <zorder>pushButton_21</zorder>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Afficher client</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_28">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>-50</y>
         <width>1141</width>
         <height>431</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(243, 224, 219);
color:black;
border-radius:15px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</normaloff>:/new/prefix1/assets/image_2023-10-05_010035082-removebg-preview.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_33">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/new/prefix1/assets/logooo.png</normaloff>:/new/prefix1/assets/logooo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>181</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Sitka Banner</family>
         <pointsize>25</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#55aa7f;&quot;&gt;View Page&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTableView" name="tableau_cl">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>451</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Tri</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
